test/extended/networking/egressip_helpers.go

```diff
@@ -1233,7 +1233,7 @@ func sendProbesToHostPort(oc *exutil.CLI, proberPod *v1.Pod, url, targetProtocol
 		go func() {
 			defer wg.Done()
 			time.Sleep(time.Duration(n) * time.Millisecond)
-			output, err := runOcWithRetry(oc.AsAdmin(), "exec", proberPod.Name, "--", "curl", "-s", request)
+			output, err := runOcWithRetry(oc.AsAdmin(), "exec", proberPod.Name, "--", "curl", "--max-time 15", "-s", request)
```

Comment:
TL;DR: `"--max-time", "15"`

I ran:
~~~
./openshift-tests run openshift/conformance/serial --run "\[sig-network\]\[Feature:EgressIP\]"
~~~

And got:
~~~
StdErr>
curl: option --max-time 15: is unknown
curl: try 'curl --help' or 'curl --manual' for more information
~~~

Should be:
~~~
diff --git a/test/extended/networking/egressip_helpers.go b/test/extended/networking/egressip_helpers.go
index 0aa7deccd1..019bdec402 100644
--- a/test/extended/networking/egressip_helpers.go
+++ b/test/extended/networking/egressip_helpers.go
@@ -1233,7 +1233,7 @@ func sendProbesToHostPort(oc *exutil.CLI, proberPod *v1.Pod, url, targetProtocol
                go func() {
                        defer wg.Done()
                        time.Sleep(time.Duration(n) * time.Millisecond)
-                       output, err := runOcWithRetry(oc.AsAdmin(), "exec", proberPod.Name, "--", "curl", "--max-time 15", "-s", request)
+                       output, err := runOcWithRetry(oc.AsAdmin(), "exec", proberPod.Name, "--", "curl", "--max-time", "15", "-s", request)
                        // Report errors.
                        if err != nil {
                                errChan <- fmt.Errorf("Query failed. Request: %s, Output: %s, Error: %v", request, output, err)
~~~